.github/workflows/deploy.yml

```diff
@@ -107,7 +107,7 @@ jobs:
       - name: Wait for images to be available
         run: |
           while true; do
-            if docker manifest inspect opencrvs/ocrvs-auth:${{ inputs.core-image-tag }}; then
+            if docker manifest inspect ghcr.io/opencrvs/ocrvs-auth:${{ inputs.core-image-tag }}; then
```

Comment:
Resolved a merge conflict here, would appreciate a confirmation that this is now correct :)